index.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+
+        <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.9.0/css/all.css"
+                integrity="sha384-i1LQnF23gykqWXg6jxC2ZbCbUMxyw5gLZY6UiUS98LYV5unm8GWmfkIS6jqJfb4E" crossorigin="anonymous" />
+        <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap" rel="stylesheet">
+
+              
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+        <link rel="stylesheet" href="Files/style.css" />
+        <title>HTML-Forms</title>
+    </head>
+
+    <body>
+        <main> 
+            <nav>
+                <button class="sign-in-button">
+                <i class="fas fa-lock"></i>&nbsp; &nbsp; Sign In</button>
+
+                <span class="already">Already have an Intuit account?</span>
+                <br>
+                <a class="recovery-link" href="#">I forgot my User ID or Password</a>
+            </nav>
+
+            <div class="logo">
+                <a href="#"><img src="Files/Assets/intuit.PNG" alt=""></a>
+            </div>
+
+            <div class="logos-under">
+                <div id="icons-container">
+                <img src="https://plugin.intuitcdn.net/identity-authn-core-ui/1.6.5/images/mint-favicon.svg" alt="icon1" class="image-icon">
+                <img src="https://plugin.intuitcdn.net/identity-authn-core-ui/1.6.5/images/mint-text.svg" alt="icon1" class="image-icon" id="mint-icon">
+                <img src="https://plugin.intuitcdn.net/identity-authn-core-ui/1.6.5/images/quickbooks-favicon@2x.svg" alt="icon1" class="image-icon">
+                <img src="https://plugin.intuitcdn.net/identity-authn-core-ui/1.6.5/images/quick-books-text.svg" alt="icon1" class="image-icon" id="quick-books-icon">
+                <img src="https://plugin.intuitcdn.net/identity-authn-core-ui/1.6.5/images/tt-favicon@2x.svg" alt="icon1" class="image-icon">
+                <img src="https://plugin.intuitcdn.net/identity-authn-core-ui/1.6.5/images/turbo-tax-text.svg" alt="icon1" class="image-icon" id="turbo-tax-icon">
```

Comment:
Don't use images for the text.
```suggestion
           <p>
                <img src="https://plugin.intuitcdn.net/identity-authn-core-ui/1.6.5/images/mint-favicon.svg" alt="icon1" class="image-icon">
                <span class="products-text">mint</span>
                <img src="https://plugin.intuitcdn.net/identity-authn-core-ui/1.6.5/images/quickbooks-favicon@2x.svg" alt="icon1" class="image-icon">
                <span class="products-text">Quickbooks</span>
                <img src="https://plugin.intuitcdn.net/identity-authn-core-ui/1.6.5/images/tt-favicon@2x.svg" alt="icon1" class="image-icon">
                <span class="products-text">turbotax</span>
            </p>
```